libs/perun/services/src/lib/init-auth.service.ts

```diff
@@ -190,6 +190,18 @@ export class InitAuthService {
     }
   }
 
+  invalidateServiceAcess(): void {
+    // Has to be promise, bc. of NG0100.
+    Promise.resolve()
+      .then(() => {
+        this.serviceAccess = false;
+        this.setLoginScreen(true);
+      })
+      .catch(() => {
+        // This will never happen
+      });
```

Comment:
Probably not the best solution, but it works, so at least I think that this can lead to better readability.
```suggestion
    // Has to be async, bc. of ExpressionChangedAfterItHasBeenCheckedError
    timer(0).subscribe(() => {
      this.serviceAccess = false;
      this.setLoginScreen(true);
    });
```